app/src/main/java/co/runloop/influencer/fragment/ContactsFragment.java

```diff
@@ -54,21 +58,37 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
         recyclerView.setAdapter(adapter);
 
+        progressBar = root.findViewById(R.id.frag_contacts_list_progressbar);
```

Comment:
Have you considered using a binding library, such as Butterknife or DataBinding? (a short comparison can be found [here](https://hackernoon.com/android-butterknife-vs-data-binding-fffceb77ed88) if you're not familiar with them)

Reply:
I tried ButterKnife before, but i see no difference in 1 line findById or 1 line with @Bind(ID), do you think we should use it ?

Reply:
This was more a general question. Of course, there's no difference when we only have one view to bind... :)

Reply:
Why only one view ? In butterKnife we need to mark each variable with @Bind()
I can somehow understand binding annotations in ButterKnife. But i don't like things like DataBinding by Google. They moves part of logic to xml
I know this is part of MVVM pattern, but it looks too inefficiently for me

Reply:
Yes, Butterknife is also more convenient in my eyes than DataBinding. Either way, this was just a thought... If you are accustomed to manually setting up the connections, I have no problem with it :)